### PR TITLE
Add buildroot

### PR DESCRIPTION
**Pull request by chmeliik**
* OSBS-8117

It is a fedora-based image which installs all (hopefully) buildroot components.

Installing `atomic-reactor` with pip brings a slight complication in that both `docker` and `docker-py` end up getting installed as dependencies, which breaks things horribly. Temporary workaround in Dockerfile.

With this PR, OSBS-Box should become usable for container builds (again).

TODO:
* [x] Update README
* [x] Fix atomic-reactor pip installation ([here](https://github.com/containerbuildsystem/atomic-reactor/pull/1285))

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,10 +2,29 @@
...
 
-## How-to
+Run a simple container build on your OSBS-Box:
+
+```bash
+$ # Get URL of docker registry
+$ OSBS_REGISTRY=$(oc -n osbs-registry get route osbs-registry --output jsonpath='{ .spec.host }')
+
+$ # Copy base image for container build to docker registry
+$ skopeo copy docker://registry.fedoraproject.org/fedora:30 \
```
Comment (MartinBasti) [inline README.md]: skopeo doesn't copy manifests-lists, we use skopeo-lite to copy image with manifests list

**Comment by MartinBasti** [comment]
Works fine :+1:

File: README.md (hunk 1 above)
Comment (MartinBasti) [reply README.md]: You can address it in separate PR.

**Comment by StykMartin** [comment]
@chmeliik
Next time you can open is as draft first.
https://github.blog/2019-02-14-introducing-draft-pull-requests/

File: README.md (hunk 1 above)
Comment (chmeliik) [reply README.md]: Addressed

File: README.md [hunk 2]
```diff
@@ -83,12 +104,65 @@ the koji certificates directory (__~/.local/share/osbs-box/certificates/koji/__
 There is one for each koji user (by default, only _kojiadmin_ and _kojiosbs_ are users, but
 logging in creates a user automatically).
 
-#### Koji CLI (local)
+### Koji CLI (local)
 
 Coming soon<sup>TM</sup>
 
+### Docker registry
```
Comment (MartinBasti) [inline README.md]: `s/docker/container/`

File: README.md [hunk 3]
```diff
@@ -83,12 +104,65 @@ the koji certificates directory (__~/.local/share/osbs-box/certificates/koji/__
...
+
+Regular `skopeo` does not copy manifest lists. Builds may work even with base images copied using
+`skopeo`, but they will not use OSBS features related to manifest lists.
+
+For this purpose, OSBS-Box provides a `skopeo-lite` image.
+
+Use it with `podman`:
+
+```bash
+$ podman build skopeo-lite/ --tag skopeo-lite
```
Comment (MartinBasti) [inline README.md]: I think we can make this easier for people and automatically builds that image in https://quay.io/organization/containerbuildsystem namespace

File: README.md (hunk 1 above)
Comment (twaugh) [reply README.md]: FWIW I believe [skopeo v0.1.40](https://github.com/containers/skopeo/releases/tag/v0.1.40) adds a `--all` flag to do just that.

File: README.md (hunk 2 above)
Comment (chmeliik) [reply README.md]: Do I `s/docker/container/` for every single instance where the registry is mentioned? Or just this title? For what it's worth, it _is_ a __docker__ registry.

File: README.md (hunk 3 above)
Comment (chmeliik) [reply README.md]: It appears that I do not have any of the permissions needed to do that, best I can do is create a repo in my quay.io namespace and build it from my forked osbs-box repo.

**Comment by chmeliik** [comment]
Changed docker registry => container registry, `skopeo-lite` is now considered a fallback option.

**Comment by chmeliik** [comment]
Drop "Temporary fix for atomic reactor installation"

**Comment by StykMartin** [comment]
:rocket:

**Pull request merged**